requirements/base.txt

```diff
@@ -20,7 +20,8 @@ django-debug-toolbar==3.2
 newspaper3k==0.2.8
 
 git+git://github.com/DemocracyClub/dc_base_theme.git@0.3.11
-git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.0
+git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.
```

Comment:
Think a trailing `0` was removed by mistake here which is causing the CI check to fail